snuba/web/views.py

```diff
@@ -50,7 +50,7 @@
 
 class QueryResult(NamedTuple):
     # TODO: Give a better abstraction to QueryResult
-    result: ClickhouseQueryResult
+    result: RawQueryResult
```

Comment:
I think these two names together now sound confusing, before the responsibilities were more clear.
Would it make sense to make QueryResult -> HTTPQueryResult ? Since essentially it adds the http status code.

Reply:
Going to rename this `WebQueryResult` to match the module name and avoid confusion with the HTTP reader (or the possible implication that this is an actual HTTP response) if that works.

Reply:
Just realized this type definition doesn't actually make sense anyway. Ugh.

https://github.com/getsentry/snuba/blob/3ba41dc887f2f26862ad8d694d04a94106900211/snuba/web/views.py#L285-L301

Going to give this some thought.

Reply:
This is about as far as I'm willing to go with this for now:

https://github.com/getsentry/snuba/blob/9378bb80933cf511f91a677a4a1c0fb56ffdb71b/snuba/web/views.py#L50-L53